From 5dde234b2d3caf067ea827858c53adc5d4c56c13 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sat, 18 Jan 2014 13:56:54 -0500
Subject: [PATCH] imlib2-config: delete old reference to @my_libs@

This was cleaned up a while ago, but this file was missed.

URL: https://bugs.gentoo.org/497894
---
 imlib2-config.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imlib2-config.in b/imlib2-config.in
index cf814c6..965f7c3 100644
--- a/imlib2-config.in
+++ b/imlib2-config.in
@@ -46,7 +46,7 @@ while test $# -gt 0; do
       ;;
     --libs)
       libdirs=-L@libdir@
-      echo $libdirs -lImlib2 @my_libs@
+      echo $libdirs -lImlib2
       ;;
     *)
       echo "${usage}" 1>&2
-- 
1.8.4.3

